media-libs/osl/osl-1.9.5.recipe

```diff
@@ -0,0 +1,154 @@
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	osl$secondaryArchSuffix = $portVersion
+	cmd:oslc$commandSuffix
+	cmd:oslinfo$commandSuffix
+	cmd:testrender$commandSuffix
+	cmd:testshade$commandSuffix
+	cmd:testshade_dso$commandSuffix
+	lib:liboslcomp$secondaryArchSuffix = $portVersion compat >= 1
+	lib:liboslexec$secondaryArchSuffix = $portVersion compat >= 1
+	lib:liboslnoise$secondaryArchSuffix = $portVersion compat >= 1
+	lib:liboslquery$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libtestshade$secondaryArchSuffix = $portVersion compat >= 1
+	lib:osl.imageio$secondaryArchSuffix = $portVersion compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libboost_atomic$secondaryArchSuffix
+	lib:libboost_chrono$secondaryArchSuffix
+	lib:libboost_date_time$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_serialization$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_thread$secondaryArchSuffix
+	lib:libboost_wave$secondaryArchSuffix
+	lib:libHalf$secondaryArchSuffix
+	lib:libIex_2_2$secondaryArchSuffix
+	lib:libilmimf$secondaryArchSuffix
+	lib:libIlmThread_2_2$secondaryArchSuffix
+	lib:libImath_2_2$secondaryArchSuffix
+	lib:libLLVM_5.0$secondaryArchSuffix
+	lib:libopenimageio$secondaryArchSuffix
+	lib:libpugixml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	osl${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:liboslcomp$secondaryArchSuffix = $portVersion compat >= 1
+	devel:liboslexec$secondaryArchSuffix = $portVersion compat >= 1
+	devel:liboslnoise$secondaryArchSuffix = $portVersion compat >= 1
+	devel:liboslquery$secondaryArchSuffix = $portVersion compat >= 1
+	devel:libtestshade$secondaryArchSuffix = $portVersion compat >= 1
+	devel:osl.imageio$secondaryArchSuffix = $portVersion compat >= 1
+	"
+REQUIRES_devel="
+	osl$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	gcc_syslibs_devel$secondaryArchSuffix
+	devel:libboost_atomic$secondaryArchSuffix
+	devel:libboost_chrono$secondaryArchSuffix
+	devel:libboost_date_time$secondaryArchSuffix
+	devel:libboost_filesystem$secondaryArchSuffix
+	devel:libboost_serialization$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libboost_thread$secondaryArchSuffix
+	devel:libboost_wave$secondaryArchSuffix
+	devel:libHalf$secondaryArchSuffix
+	devel:libIex_2_2$secondaryArchSuffix
+	devel:libilmimf$secondaryArchSuffix
+	devel:libIlmThread_2_2$secondaryArchSuffix
+	devel:libImath_2_2$secondaryArchSuffix
+	devel:libLLVM_5.0$secondaryArchSuffix
+	devel:libpartio$secondaryArchSuffix
+	devel:libopenimageio$secondaryArchSuffix
+	devel:libpugixml$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:bison
+	cmd:clang$secondaryArchSuffix
```

Comment:
There [aren't any](https://github.com/haikuports/haikuports/blob/master/sys-devel/llvm/llvm-5.0.0.recipe#L262) `$secondaryArchSuffix` for clang

```diff
-	cmd:clang$secondaryArchSuffix
+	cmd:clang